base/shell/explorer/trayntfy.cpp

```diff
@@ -213,7 +213,7 @@ class CTrayNotifyWnd :
 
             ::SetWindowPos(m_hwndClock,
                 NULL,
-                ptClock.x,
+                g_TaskbarSettings.bSmallIcons ? ptClock.x : ptClock.x + 5,
```

Comment:
Regarding the "5" , could you check whether, on Windows, the traybar icons stay separated the same amount even if you artificially increase the SM_CXBORDER value (window border, via the display cpl) ?

Reply:
Thanks, so using a static value seems to make sense.